train.py

```diff
@@ -1,72 +1,86 @@
 import cv2
 import numpy
+import sys
 
-from lib.utils import get_image_paths, load_images, stack_images
-from lib.training_data import get_training_data
+from utils import get_image_paths, load_images, stack_images
+from training_data import get_training_data

```

Comment:
`utils.py` is in the `lib` package, so I'm not sure if this would even work?

Reply:
Actually I merged from another version of this script and it made a lot of unwanted changes like that.

I guess it'd be better to start from scratch if you're already working on the command line arguments.